.gitignore

```diff
@@ -42,3 +42,4 @@ celerybeat-schedule.db
 .coverage
 .tox
 .cache
+*.sublime-*
```

Comment:
@bwarren2, please, comment in here so it is easy to follow up :)
> Awesome! Should I remove the .gitignore edit and force-push over this branch, or open a new PR?
We'd be better off with a new PR.